titan/features/incar.py

```diff
@@ -88,7 +88,7 @@ def update_agent(self, model: "model.HIVModel"):
                             <= self.agent.location.params.incar.haart.discontinue
                         ):
                             self.agent.haart.active = False  # type: ignore[attr-defined]
```

Comment:
might be a to do for another day, but philosophically, I don't think any feature should be directly updating another feature's status, but rather calling a method that handles the enrollment/unenrollment.  This way if we add/remove attributes, it's all in one spot.

Reply:
Yeah, I'd say "another day" on this one, but should be done.